client/my-sites/themes/current-theme/button.tsx

```diff
@@ -1,19 +1,18 @@
 import { Gridicon } from '@automattic/components';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import { Component } from 'react';
 
-export default class extends Component {
-	static displayName = 'CurrentThemeButton';
+interface CurrentThemeButtonProps {
+	name: string;
+	label: string;
+	icon: string;
+	href: string;
+	onClick: ( this: null, name: string ) => void;
```

Comment:
The first argument of `.bind` is the function context (i.e. `this` keyword).

It's not passed as an argument to the callback, so we can omit `this: null` from the declaration. It is not a parameter 😃